ci/check-basics.py

```diff
@@ -88,6 +88,20 @@ def complain(message):
         if path.startswith('demos/thirdparty/'):
             continue
 
+        if path.startswith('tools/accuracy_checker/configs') and path.endswith('.yml'):
+            if (OMZ_ROOT / path).is_symlink():
```

Comment:
This check is useless here, because you can only get here if `mode` is 100644 or 100755, i.e. the file is _not_ a symlink.

Additionally, you need to be aware that files that are recorded in Git as being symlinks will not necessarily be symlinks in the working copy. If `core.symlinks` is false (which it is on Windows by default), symlinks will be represented as regular files.

Reply:
@IRDonch please take a look. I hope that I have processed it